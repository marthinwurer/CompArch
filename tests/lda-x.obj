100 1 00001
101 1 00005
102 1 fffff
103 1 ffffb
150 1 00100
151 1 00101
152 1 00102
153 1 00103
200 1 18150
201 1 50000
202 1 50001
203 1 50002
204 1 50003
205 1 18153
206 1 50000
207 1 50fff
208 1 50ffe
209 1 50ffd
20a 1 00000
200
