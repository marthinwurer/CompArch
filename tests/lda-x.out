CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 200
200:  18150  LDXR  150  A[00000]  B[00000]  XR[000]
201:  50000  LDA   100  A[00000]  B[00000]  XR[100]
202:  50001  LDA   101  A[00001]  B[00000]  XR[100]
203:  50002  LDA   102  A[00005]  B[00000]  XR[100]
204:  50003  LDA   103  A[fffff]  B[00000]  XR[100]
205:  18153  LDXR  153  A[ffffb]  B[00000]  XR[100]
206:  50000  LDA   103  A[ffffb]  B[00000]  XR[103]
207:  50fff  LDA   102  A[ffffb]  B[00000]  XR[103]
208:  50ffe  LDA   101  A[fffff]  B[00000]  XR[103]
209:  50ffd  LDA   100  A[00005]  B[00000]  XR[103]
20a:  00000  HALT       A[00001]  B[00000]  XR[103]

Machine Halted - HALT instruction executed

Simulated time 64 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
